### ISSUE TITLE
UpdatebyQuery Rethrottle tests hangs

### ISSUE THREAD

**Issue by colings86** [elastic/elasticsearch#18744]
build URL:
http://build-us-00.elastic.co/job/es_core_master_window-2012/3196/testReport/junit/org.elasticsearch.index.reindex/RethrottleTests/testUpdateByQuery/

failure:
```
Test abandoned because suite timeout was reached.
```

reproduce URL:
```
gradle :modules:reindex:test -Dtests.seed=A5B5FA09B74893C9 -Dtests.class=org.elasticsearch.index.reindex.RethrottleTests -Dtests.method="testUpdateByQuery" -Dtests.es.logger.level=DEBUG -Dtests.security.manager=true -Dtests.nightly=false -Dtests.heap.size=894m -Dtests.jvm.argline="-server -XX:+UseConcMarkSweepGC -XX:-UseCompressedOops -XX:+AggressiveOpts" -Dtests.locale=da-DK -Dtests.timezone=Europe/Isle_of_Man
```

**Comment by colings86** [comment]
@nik9000 are you able to look into this?

**Comment by tlrx** [comment]
This should be resolved by https://github.com/elastic/elasticsearch/pull/18731

**Comment by nik9000** [comment]
> This should be resolved by #18731

Thanks. I've updated the PR to mark it as closing this.
